openlibrary/plugins/worksearch/languages.py

```diff
@@ -41,21 +42,40 @@ def process_key(self, key):
         return key.replace("_", " ")
 
 
+def get_top_languages(limit):
+    from . import search
+    result = search.get_solr().select('*:*', rows=0, facets=['language'], facet_limit=limit)
```

Comment:
```suggestion
    result = search.get_solr().select(
        '*:*', rows=0, facets=['language'], facet_limit=limit
    )
```
Placate `make lint-diff`